ml-agents/mlagents/trainers/policy.py

```diff
@@ -1,9 +1,12 @@
 import logging
 import numpy as np
+import tensorflow as tf
```

Comment:
@vincentpierre - I thought we were aiming to remove TF dependency from the Policy class?